wg-code-quality/wg-charter.md

```diff
@@ -0,0 +1,59 @@
+# WG Code-Quality Charter
+
+## Scope
+
+The Code-Quality Working Group has two goals:
+
+* (short-term) **focus on the improvement of code quality** and
+* (long-term) **create an inter-SIG process for identifying, agreeing on and addressing code quality issues**
```

Comment:
For me, line 14 is summarizing this and describe why the WG is needed.
The long-term goal here, which mentions a processes, is not very clear to me.
I would expect the WG to have artifacts from which the quality will be improved and sustained over time.

This may be partially achieved through processes, but I am unsure if that is the "solution" or how it will look-like.

Reply:
I have changed the In Scope section by changing the last sentence to what you pointed out before, which IMHO brings it down into an actionable goal. WDYT?

Reply:
I guess I do not see this inter-SIG process in my mind and therefore have hard time to understand and agree on it being a goal. I feel this section is redundant as everything is described in the next section.

However, this is a small part compared to all this charter content, so I should not get stuck on it.
We can raise it later and adjust per need.